lua/plenary/path.lua

```diff
@@ -340,7 +340,7 @@ function Path:normalize(cwd)
   self:make_relative(cwd)
 
   -- Substitute home directory w/ "~"
-  self.filename = self.filename:gsub("^" .. path.home, "~", 1)
+  self.filename = self.filename:gsub("^" .. path.home, "~/", 1)
```

Comment:
We should probably use `path.sep` here, to handle Windows too

Reply:
It's a good idea. Unfortunately, I don't have any Windows machine to test this. CI doesn't run on Windows host as well. For these reasons, I didn't alter tests to use `path.sep`. I would be grateful if somebody with Windows could try it.

Reply:
I have done some (admittedly, limited) testing on my Windows machine, and things seem to work as expected 🙂